examples/show-toast/stripe-app.json

```
@@ -0,0 +1,25 @@
+{
+  "id": "dev.t6d.toaster",
+  "version": "0.0.1",
```

```suggestion
  "version": "0.1.0",
```
Just for consistency with the `package.json`.